profile/README.md

```diff
@@ -21,26 +21,26 @@ Scratch Addons is not affiliated, associated, or in any way officially connected
 
 ### 🔧 Browser extension
 
-**GitHub repository:** [ScratchAddons/ScratchAddons](https://github.com/ScratchAddons/ScratchAddons) | [README](https://github.com/ScratchAddons/ScratchAddons#readme) | [Issues](https://github.com/ScratchAddons/ScratchAddons/issues) | [Pull requests](https://github.com/ScratchAddons/ScratchAddons/pulls) | [Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions)<!-- 2 spaces -->  
+**GitHub repository: [📁 ScratchAddons/ScratchAddons](https://github.com/ScratchAddons/ScratchAddons) | [🪧 README](https://github.com/ScratchAddons/ScratchAddons#readme) | [📝 Issues](https://github.com/ScratchAddons/ScratchAddons/issues) | [💫 Pull requests](https://github.com/ScratchAddons/ScratchAddons/pulls) | [👥 Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions)**<!-- 2 spaces -->  
```

Comment:
The README emoji is too new. I can't see it on Windows 10

Reply:
It's a signpost. iOS and Windows 11 have it, but I guess Windows 10 is still missing that emoji.

Reply:
Can you change it to another emoji?

Reply:
Alright.